drip-woocommerce.php

```diff
@@ -1,20 +1,25 @@
 <?php
 /**
- * Plugin Name: Drip WooCommerce
- *
  * @package Drip_Woocommerce
  */
+/*
+Plugin Name: Drip WooCommerce
+Plugin URI: https://github.com/DripEmail/drip-woocommerce
+Description: A Wordpress plugin to connect to Drip's WooCommerce integration
+Version: 0.0.1
+Author: DripEmail
+Author URI: https://github.com/DripEmail/
+License: GPLv2 or later
```

Comment:
I think we specifically do not want to say "or later." in this.  Wordpress is GPLv2 explicitly, and GPLv3 is considered a poison pill of sorts.

Reply:
Fine with me. I just copied this over from the readme. Should we change it there as well?

Reply:
I updated both spots.